File: 3. Linked List/12_create_linked_list.java
import java.util.List;

class Node<T> {
    T val;
    Node<T> next;

    public Node(T val) {
        this.val = val;
        this.next = null;
    }
}

class Source {
    public static <T> Node<T> createLinkedList(List<T> values) {

        if (values.isEmpty()) {
            return null;
        }
        Node<T> head = new Node<>(values.get(0));
        Node<T> current = head;

        for (int i = 1; i < values.size(); i++) {
            current.next = new Node<>(values.get(i));
            current = current.next;
        }

        return head;
    }

}

File: 3. Linked List/12_create_linked_list.py
class Node:
    def __init__(self, val):
        self.val = val
        self.next = None

def create_linked_list(values):
    if not values:
        return None

    head = Node(values[0])
    current = head

    for value in values[1:]:
        current.next = Node(value)
        current = current.next

    return head
